spec/views/hyrax/admin/admin_sets/_form_participant_table.html.erb_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'spec_helper'
+
+RSpec.describe 'hyrax/admin/admin_sets/_form_participant_table.html.erb', type: :view do
+  let(:template) { stub_model(Hyrax::PermissionTemplate) }
+  let(:user) { create(:user) }
+  let(:access_grant) do
+    stub_model(Hyrax::PermissionTemplateAccess,
+               agent_type: 'user',
+               agent_id: user.user_key,
+               access: 'manage')
+  end
+  let(:pt_form) do
+    instance_double(Hyrax::Forms::PermissionTemplateForm,
+                    model_name: template.model_name,
+                    to_key: template.to_key,
+                    access_grants: [access_grant])
+  end
+
+  before do
+    @form = instance_double(Hyrax::Forms::AdminSetForm,
+                            to_model: stub_model(AdminSet),
+                            permission_template: pt_form)
+    # Ignore the delete button link
+    allow(view).to receive(:admin_permission_template_access_path).and_return("/admin/permission_template_accesses/123")
+  end
+
+  describe "Manager participants table" do
+    before do
+      allow(view).to receive(:access).and_return("managers")
+      allow(view).to receive(:filter).and_return(:manage?)
+      render
```

Comment:
I see what's up.  The invocation in code is here: https://github.com/projecthydra-labs/hyrax/blob/c6cd4de936f0691e29746deca69c12732d5d7bd5/app/views/hyrax/admin/admin_sets/_form_participants.html.erb#L61
so instead of stubbing these, pass them as locals on this line:

`render 'admin/admin_sets/form_participants', access: "manage", filter: :manage?`